MIMIC-IV_Example/joint_script.sh

```diff
@@ -36,9 +36,17 @@ MIMICIV_RAW_DIR="$1"
 MIMICIV_PREMEDS_DIR="$2"
 MIMICIV_MEDS_DIR="$3"
 N_PARALLEL_WORKERS="$4"
+DO_UNZIP="${5:-do_unzip=false}"  # Default to 'do_unzip=false' if not provided, this uses more disk space but reduces memory usage
```

Comment:
You need to change one more thing here -- this script is set up so that the first 4 args are mandatory, and all others are sent to the scripts as arguments via the `"$@"` thing. Right now, your `DO_UNZIP` will absorb all of those other kwargs. You should either:

1. Make sure that `DO_UNZIP` only absorbs the fifth argument if the arg there starts with `--do_unzip` or something.
- or -
2. Make there be a mandatory 5th arg that is `DO_UNZIP` and then replace the `shift 4` with `shift 5` so that it isn't used again below in the `"$@"`